index.html

```diff
@@ -21,7 +21,7 @@
   <link
     href="https://fonts.googleapis.com/css2?family=Manrope:wght@200&family=Montserrat:ital,wght@1,200&family=Open+Sans:wght@700&family=Roboto&display=swap"
     rel="stylesheet">
-  <title>Boldo</title>
+  <title>Boldo web</title>
```

Comment:
This looks more professional
```suggestion
  <title>Boldo | Welcome to Boldo</title>
```